ls1intum/Artemis
# Enhancement/guided tour/multipage navigation

### Checklist
- [x] Client: I updated multiple integration tests (Jest) related to the features

### Motivation and Context
Follow up for #1225 
This PR should fix the navigation dots display for multi-page tutorials.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Administration
3. Start or restart tutor assessment tour
4. The navigation and the displayed navigation dots should be working correctly
